builtin/game/features.lua

```diff
@@ -12,9 +12,9 @@ core.features = {
 
 function core.has_feature(arg)
 	if type(arg) == "table" then
-		missing_features = {}
-		result = true
-		for ft, _ in pairs(arg) do
+		local missing_features = {}
+		local result = true
+		for ftr in pairs(arg) do
```

Comment:
@rubenwardy the second return value would always be `true`, no need to store it in a temporary variable.